### PR TITLE
#130 log rotation

### PR DESCRIPTION
**Pull request by tomster**
unifies the log handling and limits it to 50mb to avoid filling up the card

### CODE REVIEW

File: deployment/roles/hub-backend/templates/supervisor.conf [hunk 1]
```diff
@@ -9,8 +9,12 @@ directory={{ backend_deploy_location }}
 {% endif %}
 autostart=true
 autorestart=true
-stdout_logfile={{ backend_data_location }}/wsgi.log
-redirect_stderr=true
+stdout_logfile={{ backend_data_location }}/senic_hub.log
```
Comment (larsblumberg) [inline deployment/roles/hub-backend/templates/supervisor.conf]: +1 for renaming the log file to `senic_hub.log`

**Comment by grunskis** [comment]
@tomster Until now all logs were written to a single file. I noticed only now that you introduced another file for `error` logging. Why there's need for that?

**Comment by tomster** [comment]
@grunskis it's a technical reason. since we want to use syslog, we will need to hook into supervisor's event machinery and the docs say "Setting redirect_stderr=true in an [eventlistener:x] section is now disallowed because any messages written to stderr would interfere with the eventlistener protocol on stdout"

**Comment by tomster** [comment]
but now that i re-read that, i think it doesn't actually apply in this case, since we're not setting the logging in the listener, but rather in the application.

so there's no reason not to keep it

**Pull request merged**